sections/shared/Layout/SideNav/MobileSideNav.tsx

```diff
@@ -0,0 +1,70 @@
+import styled from 'styled-components';
+import { FC } from 'react';
+import Link from 'next/link';
+import { Svg } from 'react-optimized-image';
+
+import StakingLogo from 'assets/svg/app/staking-logo-small.svg';
+import BackIcon from 'assets/svg/app/back.svg';
+import CloseIcon from 'assets/svg/app/close.svg';
+
+import SideNavContainer from 'containers/SideNav';
+import ROUTES from 'constants/routes';
+import { MOBILE_SIDE_NAV_WIDTH, zIndex } from 'constants/ui';
+
+import SideNav from './SideNav';
+import SubMenu from './MobileSubMenu';
+
+const DesktopSideNav: FC = () => {
+	const {
+		closeMobileSideNav,
+		clearSubMenuConfiguration,
+		isShowingSubMenu,
+		isShowingMobileSideNav,
+	} = SideNavContainer.useContainer();
+
+	return (
+		<Container data-testid="sidenav" isShowing={isShowingMobileSideNav}>
+			<StakingLogoWrap>
+				{isShowingSubMenu ? (
+					<Svg src={BackIcon} onClick={clearSubMenuConfiguration} />
+				) : (
+					<Link href={ROUTES.Home}>
+						<>
+							<Svg src={StakingLogo} />
```

Comment:
Thinking maybe we could add some padding to this. Kinda hard to tap the close button sometimes on mobile.

Reply:
Gotcha. Done.